src/main/java/org/jitsi/videobridge/Channel.java

```diff
@@ -800,15 +800,21 @@ public void setTransport(IceUdpTransportPacketExtension transport)
      */
     public void touch(ActivityType activityType)
     {
-        lastActivityTime.increase(System.currentTimeMillis());
+        long now = System.currentTimeMillis();
 
         switch (activityType)
         {
+            case PAYLOAD:
+                lastActivityTime.increase(now);
+                lastTransportActivityTime.increase(now);
+                lastPayloadActivityTime.increase(now);
+                break;
             case TRANSPORT:
-                touchTransport();
+                lastActivityTime.increase(now);
+                lastTransportActivityTime.increase(now);
                 break;
-            case PAYLOAD:
-                touchPayload();
+            default:
+                lastActivityTime.increase(now);
```

Comment:
Maybe merge the three lastActivityTime.increase(now) calls outside the "switch"?